src/main/java/com/desiato/puresynth/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.desiato.puresynth.exceptions;
+
+public class UserNotFoundException extends RuntimeException{
```

Comment:
There is also a Spring exception `EntityNotFoundException`. Do you want to create a specialized exception for all missing entities in your service?